qualification/antenna_channelised_voltage/test_gain.py

```diff
@@ -158,7 +162,9 @@ async def test_gains_capture_start(
 
     pdf_report.step("Capture and verify output")
     await pcc.request("capture-start", "baseline-correlation-products")
-    _, data = await receiver.next_complete_chunk(min_timestamp=0)
+    # We use dsim_timestamp as a minimum to ensure that we're not receiving
+    # data from a *previous* capture-start/stop.
+    _, data = await receiver.next_complete_chunk(min_timestamp=dsim_timestamp)
```

Comment:
Are you saying that each successive test is guaranteed to have dsim output timestamps increasing?
* How is that ensured? You can just point me to something/a file.

Reply:
Not sure I follow - time only runs forwards (there is the special case that the timestamps roll over after 2^48 samples, but that takes longer than the entire qualification test run, never mind the duration of each subarray product).

Reply:
I was asking mainly in case something was being mocked out internally to force/ensure some expected timestamp value(s). Either way, noted.